web-app/models/moisture.js

```diff
@@ -0,0 +1,17 @@
+const Sequelize = require("sequelize");
+const sequelize = require("../app.js");
+
+Moisture = sequelize.define("moisture", {
+  id: {
+    type: Sequelize.INTEGER,
+    autoIncrement: true,
+    allowNull: false,
+    primaryKey: true,
+  },
+  moisture: {
+    type: Sequelize.INTEGER,
+    allowNull: false,
+  },
+});
```

Comment:
This field is missing its associations. They should be defined here as a property "associate:", then they can be handled automatically by lines 28-32 on the `index.js` from Sequelize.